test/unit/test_autosubmit_helper.py

```diff
@@ -18,10 +18,11 @@
 """Test file for ``autosubmit_helper.py``."""
 
 import datetime
-import pytest
 from datetime import timedelta
 from typing import Callable
+import pytest
 
+from mock import MagicMock
```

Comment:
You can use the `mocker` fixture directly and let pytest-mock control mocks (useful if they ever switch lib, then we don't have to chase what's using `pytest-mock` and what's using `mock`).